### DISCUSSION TITLE
Bilibili-Evolved一直提醒我要更新

### DISCUSSION THREAD

**Discussion by zhenzhongluo** [the1812/Bilibili-Evolved#1335] [提问 / Q&A]
TemperMonkey只有1.15版本，怎样都更新不到1.17

**Comment by Coulomb-G** [comment]
1.11.5是正式版,1.11.7是预览版,你装的是哪个?另外请提供详细的反馈信息

**Comment by the1812** [comment]
greasyfork那装的话要删掉, 去这里[主页](https://github.com/the1812/Bilibili-Evolved)装

**Comment by zhenzhongluo** [comment]
其实哪个版本无所谓，只是那个每开一个页面就提示我版本过低要我升级有点郁闷而已。就是想怎么解决这个问题呢。。

Grant Howard <notifications@github.com> 于2020年12月18日周五 上午2:39写道：

> greasyfork那装的话要删掉, 去这里主页 <https://github.com/the1812/Bilibili-Evolved>装
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/the1812/Bilibili-Evolved/discussions/1335#discussioncomment-221152>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABBK5UFE3BFXVGBQ2YZ2LN3SVMBKJANCNFSM4VAYLWAQ>
> .
>


-- 
Zhen-Zhong Luo
Cell: 202-329-3414
Email: zhenzhongluo@gmail.com
